<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SteelDefectGUI</class>
 <widget class="QMainWindow" name="SteelDefectGUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SteelDefectGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="open_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 불러오기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exec_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>실행하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>종료하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>이미지 목록</string>
    </property>
   </widget>
   <widget class="QListWidget" name="IMG_list_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="origin_img_view">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>800</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>원본 이미지</string>
    </property>
   </widget>
   <widget class="QLabel" name="pre_img_view">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>800</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>전처리 후 이미지</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>결함 종류 분포</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_view">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>270</y>
      <width>340</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>50</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>결함 목록</string>
    </property>
   </widget>
   <widget class="QListWidget" name="defect_view_widget">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>70</y>
      <width>340</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
